waypoint_manager: {
#NEU inertial frame
waypoints:
 [[0.0, 0, 2.0, 0],
#  [1.0, 0, 4.0, 1.0],
#  [0, 4.5, 7.0, 1.0],
#  [4.5, -4.5, 4.5, 3.14],
 [-0.8, -1.0, 2.0, 0.]],

threshold: 0.3,
#TODO: need to come up with a better way to have a threshold.  Should be a function of velocity or something.
rendevous_threshold: 0.3,
center_threshold: 0.1,
landing_threshold: 0.1,
landing_orient_threshold: 10,
cycle: True,
auto_land: False,
begin_descent_height: 2.0,
begin_landing_height: 0.3,
antenna_offset: [0.0, 0.0, 0.0],
# antenna_offset: [0.8, 0.65, -0.25],
}
